cdm/src/main/java/ucar/nc2/dataset/conv/CF1Convention.java

```diff
@@ -135,7 +135,19 @@ public void augmentDataset(NetcdfDataset ds, CancelTask cancelTask) throws IOExc
           v.addAttribute(new Attribute(_Coordinate.AxisType, AxisType.Ensemble.toString()));
           continue;
         }
-
+        
+        //sjk code
+        if (v.findAttribute(CF.GEOMETRY) != null) {
+        	v.addAttribute(new Attribute(CF.NODES, v.findAttIgnoreCase(CF.NODES, "nodes")));
+        	v.addAttribute(new Attribute(CF.NODE_COUNT, v.findAttIgnoreCase(CF.NODE_COUNT, "node_count")));
+        	v.addAttribute(new Attribute(CF.NODE_COORDINATES, v.findAttIgnoreCase(CF.NODE_COORDINATES, "node_coordinates")));
+        	v.addAttribute(new Attribute(CF.PART_NODE_COUNT, v.findAttIgnoreCase(CF.PART_NODE_COUNT, "part_node_count")));
+        	if (v.findAttValueIgnoreCase(CF.GEOMETRY_TYPE, "geometry_type").equalsIgnoreCase("polygon")) {
+            	v.addAttribute(new Attribute(CF.INTERIOR_RING, v.findAttIgnoreCase(CF.INTERIOR_RING, "interior_ring")));
+        	}
+        	continue;
+        }
+        

```

Comment:
Just a really small thing but perhaps this should be put in a separate section of augmentDataset? Currently it's under "look for special standard names" but this block is looking for geometry attributes.